dashboard/app/models/concerns/pd/form.rb

```diff
@@ -39,7 +44,7 @@ def validate_required_fields
     # empty fields may come about when the user selects then unselects an
     # option. They should be treated as if they do not exist
     hash.delete_if do |_, value|
-      value.empty?
+      value.blank?
```

Comment:
I encountered an error at some point during development because I had a nil (rather than empty) value. I don't remember how I got there, but `blank?` covers `nil` and `empty?`.

Is there a reason we'd want to preserve nil values?